narratives/ncov_sit-rep_ja_2020-03-13.md

```diff
@@ -155,30 +155,30 @@ COVID-19とそれを引き起こすウイルスであるSARS-CoV-2を理解す
 <br><br>
 - ヨーロッパ、米国の一部、中国、および東南アジアでの地域拡散の証拠があります。 
 <br><br>
-- 地域拡散をコントロールするため、社会的な距離を保ち、弱者を保護することが重要です。
+- 地域拡散をコントロールするため、社会的な距離を保ち、感染の影響を受けやすい人々を保護することが重要です。
 
 <!-- This is the right-side text -->
 
 ```auspiceMainDisplayMarkdown
 # 私達に出来ること  
 
-社会的な距離、つまり毎日出会う人の数を減らすことは時には困難を伴いますが、公共の利益にとって非常に有益です。もし全員が毎日の接触数を25％減らすと、来月の累積接触件数は50％減ると予想されます。([Klein et al., 2020-03-13](https://institutefordiseasemodeling.github.io/COVID-public/reports/Working%20paper%20%E2%80%93%20model-based%20estimates%20of%20COVID-19%20burden%20in%20King%20and%20Snohomish%20counties%20through%20April%207.pdf)). 
-社会的距離についてはこちらを参照して下さい。[Check out this helpful guide](https://www.theatlantic.com/family/archive/2020/03/coronavirus-what-does-social-distancing-mean/607927/).
+社会的な距離、つまり毎日出会う人の数を減らすことは時には困難を伴いますが、公共の利益にとって非常に有益です。もし全員が毎日の接触数を25％減らすと、来月の累積症例数は50％減ると予想されます。([Klein et al., 2020-03-13](https://institutefordiseasemodeling.github.io/COVID-public/reports/Working%20paper%20%E2%80%93%20model-based%20estimates%20of%20COVID-19%20burden%20in%20King%20and%20Snohomish%20counties%20through%20April%207.pdf)). 
+社会的距離については[こちら](https://www.theatlantic.com/family/archive/2020/03/coronavirus-what-does-social-distancing-mean/607927/)を参照して下さい。
 <div>
   <img src="https://github.com/nextstrain/ncov/raw/master/figures/social-distancing-efficacy.png" width="70%">
 </div>
 
 ## 個人が出来ること
-* 特に脆弱なグループ（高齢者や既往症のある方など）の場合は、毎日接触する人数を減らしてください。
-* あなたが健康であっても、あなたの周りの多くの人々がそうでないことを忘れないでください。 他の人を守るためにこれらの慣行に従ってください。
-* 良く手を洗って下さい。(Translator note: Is there a Japanese joke that changes to "like you just chopped a jalapeno and have to change a contact lens"? :) ) 
-* 体調が悪い場合は家にいましょう。 自主検疫（自主隔離）する必要がある場合に備えて、いくつかの追加の備品を用意しておきましょう。
+* 特に感染の影響を受けやすい集団（高齢者や既往症のある方など）の場合は、毎日接触する人数を減らしてください。
+* あなたが健康であっても、あなたの周りの多くの人々がそうでないことを忘れないでください。 他の人を守るために次に上げるこれらの慣行に従ってください。
+* 良く手を洗って下さい。(辛いものを触ったあとにコンタクトレンズを変えなければならないように) 
```

Comment:
IMO, jalapeno is a metaphor of something spicy
So I translated it as `something spicy` for now.
You may replace it with an example of something spicy

cc @fzhang12